otherlibs/unix/fork.c

```diff
@@ -38,7 +38,7 @@ CAMLprim value caml_unix_fork(value unit)
   int ret;
   if (caml_domain_is_multicore()) {
     caml_failwith
-      ("Unix.fork may not be called while other domains were created");
+      ("Unix.fork may not be called after any domain was spawned");
```

Comment:
```suggestion
      ("Unix.fork may not be called after any domain has been spawned");
```

is a more correct sounding perfect tense here.

Reply:
Thanks